client/src/components/ProfileSidebar.tsx

```diff
@@ -37,8 +42,70 @@ export const ProfileSidebar = ({ traineeId }: ProfileSidebarProps) => {
       paddingY={4}
     >
       {/* Profile image */}
-      <Box height={180} width={180} display="flex" justifyContent="center">
-        <Avatar variant="square" sx={{ width: '100%', height: '100%' }} src={data?.imageURL} alt={data?.displayName} />
+      <Box
+        onMouseEnter={() => {
+          setIsHovering(true);
+        }}
+        onMouseLeave={() => {
+          setIsHovering(false);
+        }}
+        height={180}
+        width={180}
+        display="flex"
+        justifyContent="center"
+        position="relative"
+      >
+        <Avatar
```

Comment:
Maybe it is a good idea to start thinking about extracting the profile picture to it's own component so it will not clutter the sidebar component